LoopKitUI/View Controllers/TitleTableViewController.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  TitleTableViewController.swift
+//  LoopKitUI
+//
+//  Created by Nathaniel Hamming on 2020-01-27.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import UIKit
+
+open class TitleTableViewController: UITableViewController {
+
+    public var tableViewTitle: String?
+    
+    open override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.register(SectionHeaderView.self, forHeaderFooterViewReuseIdentifier: SectionHeaderView.reuseIdentifier)
+        tableView.sectionFooterHeight = UITableView.automaticDimension
+        tableView.estimatedSectionFooterHeight = 10
+        tableView.sectionHeaderHeight = UITableView.automaticDimension
+        tableView.estimatedSectionHeaderHeight = 70
+    }
+    
+    override open func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        guard section == 0 else {
+            return nil
+        }
+        return tableViewTitle
+    }
+    
+    override open func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        guard let headerTitle = self.tableView(tableView, titleForHeaderInSection: section) else {
+            return nil
+        }
+
+        guard let view = tableView.dequeueReusableHeaderFooterView(withIdentifier: SectionHeaderView.reuseIdentifier) as? SectionHeaderView else {
+            return nil
+        }
+
+        view.title.text = headerTitle
+        switch section {
+        case 0:
+            view.title.font = .preferredFont(forTextStyle: .title2)
+        default:
+            view.title.font = .preferredFont(forTextStyle: .headline)
```

Comment:
[nit] The default will never be hit (due to the first `guard` statement in this function). I'd do away with the entire switch.